docs/3/modules/abbreviation.yml

```
@@ -0,0 +1,14 @@
+name: abbreviation
+
+description: |-
+    This module allows commands to be abbreviated by appending a full stop. For example, executing `/PRIV. #test Hello!` would expand to `/PRIVMSG #test Hello!`.
+
+    If more than one command matches the specified abbreviation the server will respond with a list of commands that the abbreviation could possibly match.
+
+configuration: null
```

Can we just omit the things that modules don't contain?

---

We could, but I'm worried it would silently ignore typos like this:

```
Configuration: |-
    blah blah
```

---

I think that's fine. Pages are usually checked before being committed anyway.

---

I know I often forget :|

But ok, done.